go/stream/message.go

```diff
@@ -59,6 +64,13 @@ type MessageProperties struct {
 	Stage                string    `json:"stage,omitempty"`                // optional
 	Compression          string    `json:"compression,omitempty"`          // optional
 	Encryption           string    `json:"encryption,omitempty"`           // optional
+	IsBot                bool      `json:"isBot,omitempty"`                // optional
```

Comment:
Can you please move `EncryptionKeyID` above `IsBot` for better readability? It would be nice to have related fields as adjacent.